README.md

```diff
@@ -2,7 +2,7 @@
 
 CentOS 8 / CentOS Stream 8 升级 CentOS Stream 9 简明教程
 
-[中文](README.md) | English(WIP)
+[中文](README.md) | [English](README_EN.md)
 
 ---
 
```

Comment:
Please help me change L9 to 
``` markdown 
哦，我的老伙计！你知道吗？`CentOS 8`在2021年12月31日完蛋啦！🎉🎉🎉  
```
Please note the two spaces at the end of the sentence.